Sources/WordPressData/Objective-C/ReaderPost.m

```diff
@@ -347,9 +347,9 @@ - (NSString *)likeCountForDisplay
     if (count == 0) {
         title = likeStr;
     } else if (count == 1) {
-        title = [NSString stringWithFormat:@"%d %@", count, likeStr];
+        title = [NSString stringWithFormat:@"%ld %@", (long)count, likeStr];
     } else {
-        title = [NSString stringWithFormat:@"%d %@", count, likesStr];
+        title = [NSString stringWithFormat:@"%ld %@", (long)count, likesStr];
```

Comment:
Xcode suggested this: to cast the `NSInteger` to long and use it via `ld` in the formatting.